app/code/Magento/ConfigurableProduct/Model/Product/Type/Configurable.php

```diff
@@ -81,7 +81,7 @@ class Configurable extends \Magento\Catalog\Model\Product\Type\AbstractType impl
      *
      * @var string
      */
-    protected $_usedProducts = '_cache_instance_products';
+    public $_usedProducts = '_cache_instance_products';
```

Comment:
Instead of making this a `public` property, we could keep this as `protected` and create a `public` getter function to return it's value instead.

```php
public function getUsedProductsKey(): string
{
    return $this->_usedProducts;
}
```